index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./main.css" rel="stylesheet">
+    <link rel="icon" href="./assets/icons/icons8-reactos.svg">
+    <title>Neon Library</title>
+</head>
+
+<body>
+    <header>
+        <nav id="navbar" class="navbar">
+            <div class="logo">
+                <img src="./assets/icons/icons8-reactos.svg" alt="logo">
+                <a href="#">
+                    <h1 class="logo-title text-xl">Neon-UI</h1>
+                </a>
+            </div>
+            <div class="search-input">
+                <input type="search" placeholder="Search docs" />
+            </div>
+            <ul class="nav-links">
+                <li>
+                    <img src="./assets/icons/github-icon.svg" class="icon-logo">
+                    <a href="#" class="text-medium">Github</a>
+                </li>
+                <li><a href="#" class="text-medium">About</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section id="home-section">
+        <div class="home-container">
+            <div class="left-section">
+                <h1>Build UI Faster with Neon.</h1>
+                <p class="text-medium mb-text">Use predefind components to build UI.<br />
+                    Spend less time on designing components.</p>
```

Comment:
Here the indentation could be improved. 
e.g.

```
<p class="text-medium mb-text">
    Use predefind components to build UI.
    <br/>
    Spend less time on designing components.
</p>
```

Reply:
resolved.